src/meson.build

```diff
@@ -18,15 +18,15 @@ executable(
     'Frontend/Widgets/ConnectBox.vala',
     'Frontend/Widgets/FilePane.vala',
     'Frontend/Widgets/OperationsPopover.vala',
-    'Frontend/Widgets/PathBarSeparator.vala',
     'Frontend/Widgets/PathBar.vala',
+    'Frontend/Widgets/FileRow.vala',
     gresource,
     dependencies: [
         dependency('glib-2.0'),
         dependency('gobject-2.0'),
-        dependency('granite', version: '>=6.0.0'),
-        dependency('gtk+-3.0', version: '>=3.22'),
-        dependency('libhandy-1'),
+        dependency('granite-7'),
+        dependency('gtk4', version: '>=3.22'),
```

Comment:
You should simply remove `, version: '>=3.22'` or set correct required `4.*.*` version depending on the current code

Reply:
@alainm23 Is there any reason you didn't fix this?